zerver/lib/types.py

```diff
@@ -1,4 +1,5 @@
 from typing import TypeVar, Callable, Optional, List, Dict, Union, Tuple, Any, Iterable
+from mypy_extensions import TypedDict
```

Comment:
Heads up: after #12939, this should now be `from typing_extensions import TypedDict`.